meta/internal/meta.proto

```diff
@@ -0,0 +1,237 @@
+package internal;
+
+//========================================================================
+//
+// Metadata
+//
+//========================================================================
+
+message Data {
+	repeated NodeInfo Nodes = 1;
+	repeated DatabaseInfo Databases = 2;
+}
+
+message NodeInfo {
+	required uint64 ID = 1;
+	required string Host = 2;
+}
```

Comment:
Should `NodeInfo` also have a bool indicating if it's running the Metastore Service?